Bottom: dc5ddc1e83a3d46cad5dc3cdf2767b7a3f08f87f
Top:    37d7483e69cc936fe29618b8b749b01b1f2d50d0
Author: scott snyder <sss@karma>
Date:   2020-02-12 11:28:06 -0500

Refresh of cmake

---

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6846010..dba8a07 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -39,7 +39,7 @@ INCLUDE_DIRECTORIES( SYSTEM ${LCFIVertex_INCLUDE_DIRS} )
 # left here for backwards compatibility
 INCLUDE_DIRECTORIES( SYSTEM ${LCFIVertex_ROOT}/vertex_lcfi ${LCFIVertex_ROOT}/boost )
 
-INCLUDE( ${ROOT_DICT_MACROS_FILE} )
+INCLUDE( MacroRootDict )
 SET( ROOT_DICT_INPUT_HEADERS
 		${PROJECT_SOURCE_DIR}/include/lcfiplus.h
 		${PROJECT_SOURCE_DIR}/include/EventStore.h
diff --git a/doc/CMakeLists.txt b/doc/CMakeLists.txt
index 894bf35..0e57931 100644
--- a/doc/CMakeLists.txt
+++ b/doc/CMakeLists.txt
@@ -63,7 +63,7 @@ IF( INSTALL_DOC )
         # --- out-of-source installations ---
         SET( DOC_INSTALL_PREFIX doc/${PROJECT_NAME} )
     
-        INSTALL( FILES release.notes CEDPicking.pdf DESTINATION ${DOC_INSTALL_PREFIX} )
+        #INSTALL( FILES release.notes CEDPicking.pdf DESTINATION ${DOC_INSTALL_PREFIX} )
 
     ENDIF()
